core/builder.go

```diff
@@ -196,18 +196,20 @@ func setupNode(ctx context.Context, n *IpfsNode, cfg *BuildCfg) error {
 		opts.HasBloomFilterSize = 0
 	}
 
-	cbs, err := bstore.CachedBlockstore(ctx, bs, opts)
+	wbs, err := bstore.CachedBlockstore(ctx, bs, opts)
```

Comment:
Small nit: why is the `CachedBlockstore` renamed from `cbs` to `wbs`? Shouldn't `wbs` (which I'm not sure what it stands for) be defined later in `NewIdStore`?

Reply:
`wbs` stands for wrapped-data-store.

I could keep `cbs` and then defined yet another variable.  But that could lead to bugs where `cbs` was used when `wbs` was meant to be used.

Reply:
Ok, I was just saying because the following self-definition seem a bit weird to me,

```
wbs = bstore.NewIdStore(wbs)
```

and I would have expected to create it from another blockstore, not from itself, like

```
wbs = bstore.NewIdStore(cbs)
```

Reply:
> wbs stands for wrapped-data-store.

Shouldn't it be named `wds` instead? To avoid confusing it with a blockstore (like `cbs`).